src/main/kotlin/com/kotlindiscord/site/KDSite.kt

```diff
@@ -17,6 +19,20 @@ fun Application.main() {
 
     logger.info { "Starting KDSite version ${buildInfo.version}." }
 
+    // Connect to the database
+    connectToDb()
+
+    logger.info { "Connected to database. Applying migrations..." }
+
+    // Apply migrations
+    try {
+        MigrationsManager().migrateAll()
+
+        logger.info { "Database migrated successfully." }
+    } catch (e: IllegalStateException) {
+        logger.info { "Database already migrated." }
```

Comment:
Are we 100% sure that `IllegalStateException` is because the database is already migrated, it can be because of a divergent history or whatever?

Reply:
I'm not 100%, but it's the exception @CharlieADavies throws when that call is made.